changelog/seqera-cloud/v24.3.0_cycle4.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Seqera Cloud v24.3.0_cycle4
+date: 2025-02-26
+tags: [seqera cloud]
+---
+
+:::note
+From Wednesday 26 February 2025, Seqera’s endpoints will no longer support Windows XP and Safari versions 8 and below (including OSx 10.10 or older). This change is necessary to ensure the security and performance of our services. We recommend updating your operating system and browser version to ensure uninterrupted access to our services.
+
+Affected endpoints: 
+- cr.seqera.io
+- *.connect.cloud.seqera.io
+- intern.seqera.io
+- ai.seqera.io
+- hub.seqera.io
+- cloud.seqera.io
+- api.cloud.seqera.io
+- user-data.cloud.seqera.io
+- tower.nf
+- api.tower.nf
+- public.cr.seqera.io
+- auth.cr.seqera.io
+- wave.seqera.io
+- licenses.seqera.io
+- licman.seqera.io
+- api.multiqc.info
+:::
+
+### Feature updates and improvements
+
+- **Studios**
+  - Add support for unlimited and fully restricted running Studios quota. This change makes the quota handling consistent with other quotas and introduces negative value interpreted as disabled/fully restricted/blocked.
```

Comment:
Is it relevant to customers to add license manager and quote enforcement updates to the public changelog? This feels like internal management info, and it also implies to customers that we didn't track or enforce limits on anything before. 

Customers only need to know what their quotas are, they don't need to know how we enforce them IMO